tests/helpers.py

```diff
@@ -1572,3 +1572,32 @@ def wait_for_ct_pod_recovery():
         else:
             return False
     return True
+
+
+def label_worker_node(node_list, label_key, label_value):
+    """
+    Function to label worker node for running app pods on specific worker nodes.
+
+    Args:
+        node_list (list): List of node name
+        label_key (str): Label_key to be added in worker
+        label_value (str): Label_value
+    """
+    ocp_obj = OCP()
+    out = ocp_obj.exec_oc_cmd(
+        command=f"label node {' '.join(node_list)} {label_key}={label_value}", out_yaml_format=False
+    )
+    logger.info(out)
+
+
+def remove_label_from_worker_node(node_list, label_key):
+    """
+    Function to remove label from worker node.
+
+    Args:
+        node_list (list): List of node name
+        label_key (str): Label_key to be remove from worker node
+    """
+    ocp_obj = OCP()
+    out = ocp_obj.exec_oc_cmd(command=f"label node {' '.join(node_list)} {label_key}-", out_yaml_format=False)
```

Comment:
can you also split lines as is here :  https://github.com/red-hat-storage/ocs-ci/pull/1236/files#diff-e6ad75a5e83f9b0ee73f02b6fdcf51b5R1587-R1589

Reply:
@petr-balogh updated the code